homeassistant/components/philips_js/translations/en.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
You don't need to submit this file.

Reply:
Right. but i need it for testing. It will be overwritten once it's merged.